templates/zerver/help/move-content-to-another-stream.md

```diff
@@ -47,14 +48,15 @@ destination streams.
 
 {!message-actions-menu.md!}
 
-1. Select **Move message**. If you do not see this option, you do not have permission
-   to move this message.
+1. Select **Move messages**. If you do not see this option, you do not have permission
+   to move the message(s).
 
-1. Select the destination stream for the message from the streams dropdown list.
+1. Select the destination stream for the message(s) from the streams dropdown list.
+   If you do not have permission to move message(s) between streams, this list will be deactivated.
 
```

Comment:
Let's make this:

Select the destination stream from the streams dropdown list. If stream selection is deactivated, you do not have permission to move this message to a different stream.

Reply:
Changed a clause to "If the stream input is disabled" since "deactivated" is less precise and that felt plainer to me.